nilearn/plotting/surf_plotting.py

```diff
@@ -969,6 +970,12 @@ def plot_surf_contours(surf_mesh, roi_map, axes=None, figure=None, levels=None,
     nilearn.surface.vol_to_surf : For info on the generation of surfaces.
 
     """
+    if isinstance(figure, PlotlySurfaceFigure):
+        raise ValueError('figure argument is a PlotlySurfaceFigure'
+                         'but it should be None or a matplotlib figure')
+    if isinstance(axes, PlotlySurfaceFigure):
+        raise ValueError('axes argument is a PlotlySurfaceFigure'
+                         'but it should be None or a matplotlib axes')
```

Comment:
We should also add tests for these errors.

Reply:
Oops! Sorry. Here: c998dc7

Reply:
No worries! Thanks!